### PR TITLE
format jasmine with prettier

### PR DESCRIPTION
**Pull request by UziTech**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Format jasmine types with prettier

First part of #51825

### CODE REVIEW

**Comment by UziTech** [comment]
@chivesrs I think we need to merge this before I start on changing the types so there aren't are merge conflicts.

**Review by chivesrs — APPROVED** [review]

**Comment by chivesrs** [comment]
Seems fine to me. You'll have to wait for a DT maintainer to merge this in. Shouldn't be too long of a wait. I'd start working on your next PR but not send it out for review until this one's merged and you've rebased the commits accordingly.

**Review by djungowski — APPROVED** [review]
lgtm

**Review by sandersn — APPROVED** [review]

**Pull request merged**
